collector/configuration/store.json.md

```diff
@@ -42,4 +42,69 @@ The alias is used as a key reference for tags, units and types values.&#x20;
 
 Obs: For **OPC-UA** sources, the tag field should be written as the following pattern: `ns=<namespaceindex>;<type>=<value>`
 
-Instead of manually configuring this file, is also possible to use the remote control page [OPC Requests](./../remote-control/sources/opc-requests.md) to do this settings.
+## LiveRig Collector 5.0.0 or above
+
+Since Liverig Collector version 5.0.0, the collector can be configured to extract field from object values in **OPC-UA** sources. 
```

Comment:
```suggestion
Since LiveRig Collector version 5.0.0, the collector can be configured to extract field from object values in **OPC-UA** sources. 
```